modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/configurator/ClientExtensionProjectConfigurator.java

```diff
@@ -1089,6 +1199,41 @@ private boolean _isActiveProfile(Project project, String profileName) {
 		return false;
 	}
 
+	private boolean _isSpecialLanguageProject(File rootDir, File projectDir) {
```

Comment:
This method name needs to change. I think it's best to put it into a dedicated static util under the name `isLanguageProject` or something like that.

Reply:
Move its overloads, too.

Reply:
@drewbrokke  Where should I place that static util class? We have `internal/util`, but that feels generic. Maybe create `/internal/client/extension/util` and place it there.

Name suggestion: `LanguageClientExtensionUtil`

Reply:
I put it in  `/internal/client/extension/util`, seems to make more sense